install/helm/agones/values.yaml

```diff
@@ -94,6 +94,9 @@ agones:
       failureThreshold: 3
       timeoutSeconds: 1
     allocationBatchWaitTime: 500ms
+    replicas: 2
+    pdb:
+        minAvailable: 1
```

Comment:
remind that everything in values.yaml needs to be documented

Reply:
Added to `helm.md`